### PR TITLE
files generated from itselves

### PR DESCRIPTION
**Pull request by hemarkus**
Handle overwritten files

### CODE REVIEW

File: cmd/modules/analyzers/pkg-analyzer/main.go [hunk 1]
```diff
@@ -75,9 +75,15 @@ func (pkganalyzer *PkgAnalyzer) Analyze(controlService service.ControlServiceCli
 			}
 
 			for _, target := range pkganalyzer.targetsSlice {
-				re := regexp.MustCompile(filepath.Join(pkganalyzer.targetsDir, target))
-				if re.MatchString(fileNode.Path) {
-					pkgNode.Targets = append(pkgNode.Targets, fileNode)
+				joinedPath := filepath.Join(pkganalyzer.targetsDir, target)
+				if fileNode.Path == joinedPath {
```
Comment (GiasemiSh) [inline cmd/modules/analyzers/pkg-analyzer/main.go]: Why did you remove the regexp? If the targets contain regular expressions this will fail.

Comment (hemarkus) [reply cmd/modules/analyzers/pkg-analyzer/main.go]: Right! Will revert this change.

Comment (hemarkus) [reply cmd/modules/analyzers/pkg-analyzer/main.go]: Done. Nice catch!

**Comment by hemarkus** [comment]
please retest this
